src/utils/NetlogoWEB.py

```diff
@@ -0,0 +1,272 @@
+
+# next step is putting on the refrences on the page
+# getting the data looping thru each of the years and years as heading.
+# in press, boolean top, section.
+
+from directus_sdk_py import DirectusClient
+from bs4 import BeautifulSoup
+import os
+
+# Directus setup (Replace with the correct API URL)
+DIRECTUS_URL = "https://backend.netlogo.org"  # Base API URL, not Admin panel URL
+DIRECTUS_TOKEN = "0ylVAJfZwVbnR3YKdSlbkA8pwGIhU4Ca"
```

Comment:
You should never include an API token in a file that will be stored on Github because then it is public for anyone to see and use. Anyone could then go and delete/change everything on Directus. So, You need to:

1. Remove this from the file
2. Generate a new Directus token and make sure this one no longer works
3. Put the new Directus token in an environmental variable and import it into this file (if you are unfamiliar with this, google how to import an environmental variable into a python script)
4. Make sure the file that has the Directus token in it is included in .gitignore so it doesn't get pushed to Github

Let me know if you have any questions on any of that.